Allwmake

```diff
@@ -33,23 +38,27 @@ log "Building the OpenFOAM-preCICE adapter..."
 export ADAPTER_PREP_FLAGS
 export ADAPTER_TARGET_DIR
 
-if pkg-config libprecice; then
-  ADAPTER_PKG_CONFIG_CFLAGS="$(pkg-config --silence-errors --cflags libprecice)"
-  ADAPTER_PKG_CONFIG_LIBS="$(pkg-config --silence-errors --libs libprecice)"
+# Check if pkg-config is available and get the config for preCICE
+ADAPTER_PKG_CONFIG_CFLAGS=""
+ADAPTER_PKG_CONFIG_LIBS=""
+if command -v pkg-config >/dev/null 2>&1; then
+    if pkg-config libprecice; then
+        ADAPTER_PKG_CONFIG_CFLAGS="$(pkg-config --silence-errors --cflags libprecice)"
+        ADAPTER_PKG_CONFIG_LIBS="$(pkg-config --silence-errors --libs libprecice)"
+        log ""
+        log "If not already known by the system, preCICE may be located using:"
+        log "  pkg-config --cflags libprecice  = ${ADAPTER_PKG_CONFIG_CFLAGS}"
+        log "  pkg-config --libs libprecice    = ${ADAPTER_PKG_CONFIG_LIBS}"
+    else
+        warning "pkg-config configuration file 'libprecice.pc' was not found in the PKG_CONFIG_PATH.
+        Check the preCICE documentation page 'Linking to preCICE' for more details, or proceed if you know what you are doing."
```

Comment:
I like the new output. Only the last part 'or proceed if you know what you are doing' fits not so well in my opinion as there are only two possible scenarios: building succeeds -> should I proceed now? building  fails -> I can't really proceed.

Reply:
Good point. I had in mind that the system may know about preCICE even without pkg-config, but this is not really helpful for the user. I replaced it with "If building succeeds, ignore this warning.".